go/apps/api/openapi/spec/paths/v2/permissions/listPermissions/index.yaml

```
@@ -3,7 +3,8 @@ post:
     - permissions
   summary: List permissions
   description: |
-    Retrieve all permissions in your workspace. Results are paginated and sorted alphabetically by permission name.
+    Retrieve all permissions in your workspace. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)